components/List.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { nowPlayingMovies } from "../lib/db";
+import MovieCard from "./Card";
+
+const StyledList = styled.li`
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
```

Comment:
this is the default direction, so you can remove this line of code

Reply:
should I create an extra component for styling?

Reply:
this is already an extra component. You can keep it in the file where you use it, but you can also think about moving it someplace else and export it. Which makes a lot of sense when reusing styled components.

For the moment, it is fine to leave it where it is :)